.gitignore

```diff
@@ -1,18 +1,19 @@
 **/.terraform/**
 **tfstate*
 **/*.pem
-terraform.tfvars
```

Comment:
We should keep this

Reply:
Reinstated. And now it makes sense as they are no more committed terraform.tfvars files